src/collective/behavior/amp/utils.py

```diff
@@ -59,6 +59,7 @@ def remove_invalid_tags(self, el):
             logger.debug('<{0}> tag was removed'.format(tag.tag))
 
     def __call__(self, code):
+        code = safe_unicode(code)
```

Comment:
best to use the `raw_encoded` property [from RichTextValue](https://github.com/plone/plone.app.textfield/blob/master/plone/app/textfield/value.py#L59-L65)